react/components/molecules/key-value/key-value.stories.js

```diff
@@ -9,13 +9,15 @@ storiesOf("Atoms", module)
     .add("Key Value", () => {
         const key = text("Key", "Key");
         const value = text("Value", "Value");
+        const keyColor = color("Key Color", "#4f7af8");
+        const valueColor = color("Value Color", "#223645");
         const icon = select(
             "Icon",
             { Add: "add", Alarm: "alarm", Bell: "bell", Phone: "phone", None: null },
             null
         );
-        const iconBackgroundColor = text("Icon Background Color", "#000000");
-        const iconColor = text("Icon Color", "#ffffff");

```

Comment:
Use `text`, we're staying away from `color` because it doesn't allow entering hex.